src/auto-evo/RunResults.cs

```diff
@@ -222,7 +222,7 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                 else
                 {
                     builder.Append("  ");
-                    builder.Append(" went extinct in ");
+                    builder.Append(TranslationServer.Translate("WENT_EXTINCT_IN"));
                     builder.Append(PatchString(patch));
```

Comment:
For some languages it is not natural to have this say "went extinct in x" so this should instead use a translation string with placeholder in it that uses PatchString(path) as the parameter to format.